FND/OwnershipAndControl/Ownership.rdf

```diff
@@ -115,7 +116,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>owner</rdfs:label>
-		<skos:definition>A party in the ownership role; one that owns something. The thing owned is an Asset to that Party.</skos:definition>
+		<skos:definition>party that is legally recognized as having the ultimate control over, and right to use, something as long as the law permits and no agreement limits their rights</skos:definition>
```

Comment:
There is a very real issue with our ability and the resources to focus on some of these definitions as well - the original one was circular and not ISO 704 compliant.  The intent here was to address circularity in light of new hygiene tests that are forthcoming, and this particular definition falls into that category. We should eliminate circularity and address definitions as the opportunity arises, in other words.  There is no intent to bury this, but raising a separate issue requires some number of days lag so that one avoids github clashses, which means that it likely won't get done.

The legal definition of ownership includes the concepts of right to possession, the privilege to use, and ability to transfer any rights or privileges associated with whatever it is that one owns.  Provability of such ownership may require documentation (i.e., pedigree), depending on what one wants to do.  In general business law terms, that is the definition of ownership in the US and UK.  There are additional nuances in US law around sales or leasing of certain rights, such as water and mineral rights, that, depending on how the contracts are written, may or may not survive in a sale.  

I've rephrased to use the wording I've inserted in this note and eliminated the ambiguity at the end, given that we retain 'as long as the law permits', which would cover ownership of mineral rights.  The ability to transfer certain rights covers leasing.  If one has transferred some rights, that doesn't mean that they don't still own something, and the slightly revised definition allows for that.  Any jurisdiction-specific definition, including more specific definitions with respect to mineral rights, leasing, etc., would need to be stated on subclasses.